docs/release-notes/v2.0.rst

```diff
@@ -1,7 +1,224 @@
+.. include:: ../references.txt
+
+
 .. _gammapy_2p0_release:
 
-2.0 (unreleased)
+2.0 (August XX, 2025)
 ----------------
 
 
-- No changes yet
+- Released August XX, 2025
+- 25 contributors
+- 235 (XX)  pull requests since v1.3 (not all listed below)
+- 100 (XX) closed issues
+
+Summary
+~~~~~~~
+
+This release introduces a number of performance improvements, new features and bug fixes.
+Improved support is provided for Fermi-LAT analysis, and full analysis for VERITAS telescope is supported.
+Bayesian inference with nester sampling is now supported.
+There has been significant code cleanup, specially with respect to the documentation and the tutorial structure has
+been cleaned up. cython code for computation of fit statistics is now replaced by jit code.
+Dependence on `lru_cache` is removed.
+
+
+API changes
+~~~~~~~~~~~
+
+Two API-breaking changes have been introduced in this version:
+
+- `~gammapy.EventList.select_parameter` now takes a ``values`` and ``is_range`` argument instead of ``band`` to allow exact matches
+- ``cta_south`` and `cta_north` have been renamed as ``ctao_south`` and ``ctao_north``, resp. in `~gammapy.data.observatory_locations`
+Features deprecated since Gammapy v1.3 have been removed.
+
+Infrastructure
+~~~~~~~~~~~~~~
+
+- TODO
+
+
+Documentation improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Documentation appearance has been updated using less custom css code,
+and the ordering of tutorials restructured for clarity.
+- Four new tutorials have been added:
+   - one explaining VERITAS analysis along with addition of VERITAS data in `$GAMMAPY_DATA`
+   - one explaining computation of upper limits for non-detected sources
+   - one explaining computation of upper limits on parameters
+   - one introducing Bayesian inference with nested sampling
+- The tutorial exposing FERMI-LAT analysis has been rewritten to expose the support for FERMI analysis
+- Code examples have been added in docstrings.
+
+
+New features
+~~~~~~~~~~~~
+
+*gammapy.analysis*
+
+- Improvement of the High Level Interface using the `Analysis` class
+
+*gammapy.catalog*
+
+- Add pulse profiles for Fermi 3PC catalog
+
+*gammapy.data*
+
+- ￼Helper method for observation selection at Datastore level, specially for WCD
+
+*gammapy.datasets*
+
+- Add `Datasets.to_asimov_datasets`
```

Comment:
```suggestion
```
`_to_asimov_datasets` and `_to_asimov_dataset` are not public.
We only use them internally to compute sensitivity.